.vib/grafana-k6-operator/runtime-parameters.yaml

```diff
@@ -0,0 +1,201 @@
+            - ALL
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+    starter:
+      containerSecurityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+    runner:
+      containerSecurityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: k6-test
+  data:
+    test.js: |
+      import http from 'k6/http';
+      import { Rate } from 'k6/metrics';
+      import { check, sleep } from 'k6';
+
+      const failRate = new Rate('failed_requests');
+
+      export let options = {
+        stages: [
+          { target: 200, duration: '30s' },
+          { target: 0, duration: '30s' },
+        ],
+        thresholds: {
+          failed_requests: ['rate<=0'],
+          http_req_duration: ['p(95)<500'],
+        },
+      };
+
+      export default function () {
+        // Modified site to support airgapped
+        const result = http.get('http://nginx-vib-tests:8080');
+        check(result, {
+          'http response status code is 200': result.status === 200,
+        });
+        failRate.add(result.status !== 200);
+        sleep(1);
+      }
+# END EXAMPLE
+# EXTRAS
+# In order to support airgapped envs, we deploy a custom nginx instead of
+# the external site the original example did
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    namespace: '{{ include "common.names.namespace" . }}'
+    name: nginx-vib-tests
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: nginx
+    template:
+      metadata:
+        labels:
+          app: nginx
+      spec:
+        containers:
+        - name: nginx
+          image: bitnami/nginx
+          ports:
+          - containerPort: 8080
+          securityContext:
+            runAsNonRoot: true
+            privileged: false
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: ["ALL"]
+            seccompProfile:
+              type: "RuntimeDefault"
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: nginx-vib-tests
+    namespace: '{{ include "common.names.namespace" . }}'
+    labels:
+      app: nginx
+  spec:
+    selector:
+      app: nginx
+    ports:
+    - port: 8080
+      name: http
+      targetPort: 8080
+# As they are scratch containers, we need to deploy an extra container
+# for performing goss tests. We basically test that the metrics port
+# works, and also that the testrun was successfully executed
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    namespace: '{{ include "common.names.namespace" . }}'
+    name: vib-kubectl-test
```

Comment:
Do we need a deployment? Can we simply use a Pod or is there any VIB restriction?